openapi/components/schemas/User.yaml

```
@@ -1,5 +1,5 @@
 type: object
-description: A user of the portal
+description: A portal user
```

Technically it's a user of the API service. I usually use portal to refer to the web client.

```suggestion
description: A ROCC user
```